src/index.js

```diff
@@ -22,32 +51,37 @@ export default function css(options = {}) {
         }
       }
 
-      // Track the order that each stylesheet is imported.
-      if (!order.includes(id)) {
-        order.push(id)
-      }
-
       // Keep track of every stylesheet
       // Check if it changed since last render
+      // NOTE: If we are in transform block, we can assume styles[id] !== code, right?
       if (styles[id] !== code && (styles[id] || code)) {
         styles[id] = code
-        changes++
+        hasChanged = true
       }
 
       return ''
     },
     generateBundle(opts, bundle) {
-      // No stylesheet needed
-      if (!changes || options.output === false) {
-        return
+      const ids = []
+
+      // Determine import order of files
+      for (const file in bundle) {
+        const root = bundle[file].facadeModuleId
+        const modules = getCSSModules(root, this.getModuleInfo)
+        ids.push(...Array.from(modules))
```

Comment:
This might introduce duplicate css when multiple roots import the same css file. Would it make more sense to support outputting multiple css files or filtering out duplicates?

Reply:
Potentially. This was the only way that I could get access to the root file so that I could traverse the tree and get all imports

Reply:
I would think that support for multiple files would tie into #25 and #26 as well

Reply:
At least good enough for now!